hs_core/page_processors.py

```diff
@@ -208,7 +208,6 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                    'discoverable': discoverable,
                    'resource_is_mine': resource_is_mine,
                    'allow_resource_copy': allow_copy,
-                   'is_resource_specific_tab_active': False,
```

Comment:
Is this just cleanup you saw while in the code?  Or is this relevant to the metadata?

Reply:
This was part of the Resource Specific metadata content. It is code cleanup of the actual HTML I was modifying.